webpack.config.js

```diff
@@ -113,9 +113,9 @@ module.exports = (env, argv) => {
       warningsFilter: /export .* was not found in/
     },
     plugins: [
-      new ESLintPlugin({
-        extensions: ['ts', 'tsx', 'js', 'jsx'],
-      }),
+      // new ESLintPlugin({
+      //   extensions: ['ts', 'tsx', 'js', 'jsx'],
+      // }),
```

Comment:
Take a look at the .eslintrc.js file. It has an overrides section where you can configure different settings for different file name patterns. I don't know the syntax for basically disabling rule, but there probably is one.

Reply:
For now, I am adding in the webpack.config class, "exclude: [ 'src/lib', 'node_modules]" to exclude the file from being linted.